researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/ResearchStackDAO.java

```diff
@@ -0,0 +1,28 @@
+package org.sagebionetworks.bridge.researchstack;
+
+import android.content.Context;
+
+import org.researchstack.backbone.model.User;
+import org.sagebionetworks.bridge.android.manager.dao.SharedPreferencesJsonDAO;
+
+/**
+ * Created by jyliu on 2/9/2017.
+ */
+
+public class ResearchStackDAO extends SharedPreferencesJsonDAO {
```

Comment:
AccountDAO/ConsentDAO works with Bridge objects (android-sdk, no dependency on RS) and ResearchStackDAO works with objects in RS.

Also, I don't understand the user case for User, but it seems distinct from the Bridge concept of account